package.json

```diff
@@ -103,7 +104,7 @@
     "build": "rm -rf dist && tsc",
     "postbuild": "copyfiles -u 1 lib/**/*.ejs dist",
     "prepublishOnly": "npm run build",
-    "dev": "npm run build && ./bin/nightwatch",
+    "dev": "NIGHTWATCH_TS_NODE_DEV=true npx ts-node --swc ./bin/nightwatch",
```

Comment:
As we're moving to TypeScript, Nightwatch now includes a `tsconfig.json`. Is it possible to utilize the `ts-node` we previously added programmatically?

Reply:
The `ts-node` we added programmatically is 2-3 files deep, and Nightwatch does a lot of processing before reaching that step. And moving forward, code for all that processing is also going to be in TS and would need to be transpiled as well. So, we cannot use that implementation directly, and we also cannot move that implementation outside as the processing is still required.

Plus, this is only for dev runs, so doesn't make sense to make too many changes inside the codebase for this, as that works perfectly for the users.

Reply:
Okay, that makes sense.